.changelog/e691a51d20364875a2f95475e9c3b35a.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "e691a51d-2036-4875-a2f9-5475e9c3b35a",
+    "type": "bugfix",
+    "description": "The ErrorCode method for on service error types has been corrected to match the API model.",
+    "modules": [
```

Comment:
Thanks, I missed adding this in my change. 

Is bugfix the correct type? Since this is a minor breaking change.

Reply:
Potentially break-change, but was never really correct in the first place as it did not match service documentation. Kind of why I chose to keep it marked as a bugfix, but am open for changing that if we feel that it would be better to be under a feature version.

Reply:
There are few recent versions which correctly resolved the error to the error type. If someone started using the error code for those, they would see a change in behavior. 

This is a bug-fix which may result in change in behavior under some cases. To be safe, we should make these feature version